lib/server/chat-messages.service.ts

```diff
@@ -11,11 +11,18 @@ export function createChatMessagesService() {
 }
 
 class ChatMessagesService {
+  private address!: Address;
+
   constructor() {}
 
-  async getChatSettings(chatReferenceId: string, address: Address, question: string) {
-    const context = await this.fetchRelevantContext(address);
-    const tools = this.fetchRelevantTools(question);
+  async getChatSettings(
+    chatReferenceId: string, 
+    privyAddress: Address,
```

Comment:
why do you need to refactor this param to `privyAddress`?
- so far your changes seem to be non-essential refactors.
- Please focus on `simple` and `small` changes to focus on the requirements in the ticket. 🎟️